hack/verify-helm-repo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+for chart in "cloud-provider-azure"; do
```

Comment:
Usually I think we need only one chart? So no need for a loop. What do you think?

Reply:
You are correct that this is a bit superfluous, but I've implemented the verification script defensively, so that if any other charts are added to the repo we can easily add them to this verification script.

Reply:
I have no strong opinion. Let's keep it as it is.